### ISSUE TITLE
Bloodfen Razormaw (Skinning)

### ISSUE THREAD

**Issue by Firestoat** [slowtorta/turtlewow-bug-tracker#1393]
The Bloodfen Razormaw raptor mob in Dustwallow Marsh currently has a bugged loot table for skinning. Roughly 50% of the time, a player can attempt to skin a mob of this type and not receive a leather based loot drop. This is most likely a database link issue with some of the mobs that are spawned in, possibly related to there being more spawns of this type than would be normal for the 1.12.1 core with the custom content added to the zone.

In Game Screenshot of Mob -- https://postimg.cc/v410sLq8

TurtleWoW Mob Link -- https://database.turtle-wow.org/?npc=4356

WoWHead Classic Mob Link -- https://classic.wowhead.com/npc=4356/bloodfen-razormaw

**Comment by slowtorta** [comment]
I tested those and it's working as intended. The positioning of a mob cannot affect its loot table, it's simply impossible.

**Comment by Firestoat** [comment]
Thank you very much for looking into this. It appears there may be some confusion involving the issue I reported. In old retail WoW from 2004 through WoTLK pre cataclysm, every single mob that was tagged as being able to be skinned always gave a skinning loot drop of some sort when complete. This could be a form of leather, a hide, a dragonscale, or a quest drop associated with skinning in addition to the mob's normal leather / skinning reward.

This zone is the only one I've encountered on this private server emu core that has the situation of roughly a 50% chance of not getting a skinning related loot drop, and only with the mobs I did a report on. If the Emu core has a shortcoming that precludes a fix for the situation, that's understandable.
